src/pages/dashboard/hooks/useGenerateActivityElement.tsx

```diff
@@ -31,13 +31,6 @@ export function useGenerateActivityElement() {
   const generate = (activity: ActivityRecord) => {
     let text = trans(`activity_${activity.activity_type_id}`, {});
 
-    if (activity.activity_type_id === 4) {
-      text = text.replace(
-        ':user',
-        `${t('recurring_invoice')} :recurring_invoice`
-      );
-    }
```

Comment:
@beganovich This part of the logic is causing the sentence to read, let's say, 'Recurring Invoice created invoice 23123123,' but it should be 'John Doe created invoice 23123123.' Is there any specific reason why you added this part of the logic to replace `:user` with `${t('recurring_invoice')} :recurring_invoice`?

In the case where `activity.user.label` is falsy, we have a fallback to `t('system')`. So, there will not be an empty space for sure (`user: activity.user?.label ?? t('system')`).

Let me know your thoughts.